lib/mof/scanner.rb

```diff
@@ -144,10 +144,10 @@ def fill_queue
 	when "sint64" then @q.push [:DT_SINT64, CIM::Type.new(m)]
 	when "sint8" then @q.push [:DT_SINT8, CIM::Type.new(m)]
```

Comment:
We could certainly string split here as well to separate the two halves and remove the need for any fancier regex.